src/modelfree/score_agent.py

```diff
@@ -166,14 +168,18 @@ def default_score_config():
             'font_size': 24,
         },
     }
+    index_keys = []
     # If video_params['save_dir'] is None, and videos set to true, videos will store in a
     # tempdir, but will be copied to Sacred run dir in either case
 
+    noisy_agent_index = None              # Agent to add add action noise to
+    noisy_agent_magnitude = 1.0           # Magnitude of action noise for agent being noised
     mask_agent_index = None               # index of agent whose observations should be limited
     mask_agent_kwargs = {                 # control how agent observations are limited
         'masking_type': 'initialization',
     }
-
+    mask_agent_noise = None
+    hashable_mask_agent_kwargs = '_'.join([k + '_' + v for k, v in mask_agent_kwargs.items()])
```

Comment:
Is this needed just because it's not a top-level config parameter? Can we just make `masking_type` top-level? This feels very confusing.

Reply:
This isn't needed anymore; I'll remove it

Reply:
Yeah, I'm not really sure what to do here. The reason for this existing is that masking_type is within a dictionary rather than a top level parameter. I don't really like the pattern of making every argument we might pass to the mask_agent class a top level parameter, but I guess it might be the lesser of two evils here. (And, yes, the reason for this was that the kwargs dictionary wasn't hashable)